front/src/common-app/components/vote-options.styles.ts

```diff
@@ -40,6 +40,7 @@ export const buttonContainer = css`
 export const container = css`
   display: flex;
   flex-flow: column;
+  outline: none !important;
```

Comment:
Use this style on `subtitle` class, Should be not necessary the `!important`

Reply:
Fix